CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ Also, that release drops support for Python 3.9, making Python 3.10 the minimum
 
 * Dropped support for Python 3.9 [#2626](https://github.com/IntelPython/dpnp/pull/2626)
 * Removed the obsolete interface from DPNP to Numba JIT [#2647](https://github.com/IntelPython/dpnp/pull/2647)
+* Removed the `newshape` parameter from `dpnp.reshape` [#2670](https://github.com/IntelPython/dpnp/pull/2670)
```

Comment:
```suggestion
* Removed the `newshape` parameter from `dpnp.reshape`, which has been deprecated since dpnp `0.17.0`. Pass it positionally or use `shape=` on newer versions [#2670](https://github.com/IntelPython/dpnp/pull/2670).
```

Reply:
done